packages/core/js/models/LogLevel.js

```diff
@@ -20,7 +20,7 @@ const WARNING = "AEP_LOG_LEVEL_WARNING";
 const DEBUG = "AEP_LOG_LEVEL_DEBUG";
 const VERBOSE = "AEP_LOG_LEVEL_VERBOSE";
```

Comment:
can we rename all the constants too? e.g. AEP_LOG_LEVEL_VERBOSE -> VERBOSE
This comment applies for all enums

Reply:
I think we'd better keep those AEP information, then it should be easier to identify wrapper logs in react native console.

Reply:
If the usage looks like `LogLevel.VERBOSE` or `PrivacyStatus.AUTHENTICATED` I feel the string constant will be more consistent with the usage that way if we don't have any other limitations.

Reply:
I agree an Enum should be a better fit here. I don't know if there is any limitation to it. Can you create a GitHub issue for it? I need to figure out why we use String instead of Enum here, then we can decide what we can do later.